src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -416,6 +416,60 @@ private Locomotive performSwipe(SwipeElementDirection direction, WebElement elem
         return this;
     }
 
+
+
+
+    public WebElement swipeTo(SwipeElementDirection s, By by, int attempts){
+
+        int i;
+
+        if (isPresentWait(by)){
+            return driver.findElement(by);
+        }
+        else {
+            for (i = 0; i < attempts; i++) {
+
+                swipeCenter(s);
+                if (isPresentWait(by)) {
+                    return driver.findElement(by);
+                }
+
+            }
+
+        }
+        System.err.println("WARN: Element" + by.toString() + "does not exist!");
+        return null;
```

Comment:
I don't know how I feel about returning `null`, what would be the benefit to not failing the test here?

Reply:
well my thinking was that returning *null* should fail the test pretty fast if a test tries to interact with it thinking that it is WebElement. I can add some more error logging maybe if that'd be helpful?

Reply:
Hmm ok, I think it's fine.